charts/api-syncagent/templates/rbac.yaml

```
@@ -0,0 +1,115 @@
+{{- if .Values.rbac.create }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: '{{ template "name" . }}:leaderelection'
+rules:
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: '{{ template "name" . }}:leaderelection'
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: '{{ template "name" . }}:leaderelection'
+subjects:
+  - kind: ServiceAccount
+    name: '{{ template "name" . }}'
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: '{{ template "name" . }}:{{ .Release.Namespace }}:services'
```

I think the `services` suffix here doesn't make sense in the context of kcp-api-syncagent anymore. I think this should be:

```suggestion
  name: '{{ template "name" . }}:{{ .Release.Namespace }}:syncagent'
```